src/sdp.jl

```diff
@@ -404,14 +408,14 @@ function forward_simulations(model::SPModel,
     elseif info == "HD"
         get_u = SdpLoops.sdp_hd_get_u
     else
-        warn("Information structure should be DH or HD. Defaulted to DH")
+        @warn("Information structure should be DH or HD. Defaulted to DH")
         get_u = SdpLoops.sdp_dh_get_u
     end
 
-    build_Ux = Nullable{Function}(SDPmodel.build_search_space)
-
+    # build_Ux = Nullable{Function}(SDPmodel.build_search_space)
```

Comment:
dead comment?

Reply:
The following code produces an error at runtime
"build_Ux = Nullable{Function}(SDPmodel.build_search_space)"

Reply:
ok